appinventor/components/src/com/google/appinventor/components/runtime/Canvas.java

```diff
@@ -8,6 +8,8 @@
 
 import android.graphics.drawable.ColorDrawable;
 import android.graphics.drawable.Drawable;
+import android.util.Base64;
```

Comment:
Swap the order here so android.graphics comes before android.util

Reply:
Reordered imports